src/main/java/seedu/address/model/person/Note.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a Person's note in the address book.
+ */
+public class Note {
+
+    public static final String MESSAGE_NAME_CONSTRAINTS =
+            "Person notes can take any values";
+
+    public final String value;
```

Comment:
Is `public` the best access modifier for `value` ? As a rule of thumb start all data members as private, and depending on their usage promote them to protected, and if absolutely necessary, to public.

Reply:
@derrickchua